### PR TITLE
Fix for ITMS-90809: Deprecation of UIWebView

### PR DESCRIPTION
**Pull request by tlacroix**

### CODE REVIEW

**Comment by IntractableQuery** [comment]
@tlacroix Thanks, you saved us pulling a dependency last-minute in a soon-to-be app release :)

**Comment by tlacroix** [comment]
@IntractableQuery You're welcome! It did the trick for me too.

I forked the repository and added it to npmjs. I'll try to maintain it to a minimum so it stays compliant and compatible with Apple and Google's guidelines.

https://github.com/tlacroix/NativePageTransitions
https://www.npmjs.com/package/cordova-plugin-nativepagetransitions

**Comment by nitrique** [comment]
@tlacroix you saved my day ! thanks

File: src/ios/NativePageTransitions.m [hunk 1]
```diff
@@ -718,9 +718,9 @@ - (UIImage*) grabScreenshot {
 }
 
 - (BOOL) loadHrefIfPassed:(NSString*) href {
-  UIWebView *uiwebview = nil;
-  if ([self.webView isKindOfClass:[UIWebView class]]) {
-    uiwebview = ((UIWebView*)self.webView);
+  WKWebView *uiwebview = nil;
```
Comment (BRadHoc) [inline src/ios/NativePageTransitions.m:721]: Could we add a fallback here to still use UIWebView on older versions of iOS for backward compatibility?

Comment (nitrique) [reply src/ios/NativePageTransitions.m:721]: Apple deny upload if uiwebview is present in code, so you can use versioning on NPM

**Comment by EddyVerbruggen** [comment]
Thanks! And also thanks for testing this PR. It's now available from npm in version 0.7.0.

**Pull request merged**